rdr_service/dao/participant_dao.py

```diff
@@ -346,6 +346,38 @@ def get_biobank_ids_sample(self, session, percentage, batch_size):
             .yield_per(batch_size)
         )
 
+    def get_pid_rid_mapping(self, **kwargs):
+        sign_up_after = kwargs.get('sign_up_after')
+        sort = kwargs.get('sort') if kwargs.get('sort') == 'lastModified' else 'signUpTime'
```

Comment:
Could this potentially be simplified?

Reply:
Any suggestion?

Reply:
Sorry, never mind. I was only looking at this and line 361, I missed that you want to fix it to a specific value to send back on 370 if it wasn't 'lastModified'.